package.json

```diff
@@ -130,5 +128,17 @@
     "webpack-dev-middleware": "^1.8.4",
     "webpack-dev-server": "^2.1.0-beta.12",
     "webpack-hot-middleware": "^2.13.2"
+  },
+  "jest": {
+    "collectCoverageFrom" : ["src/**/*.{js,jsx}", "!**/node_modules/**"],
+    "testEnvironment": "node",
```

Comment:
By default this is "jsdom", which injects a "window" object into the global. But some of our tests rely on not having a "window" object. Because we're testing a webapp I think we should keep the default "jsdom" and fix the tests, but do it in a separate patch.